commands/client-kill.md

```diff
@@ -13,6 +13,7 @@ With the new form it is possible to kill clients by different attributes
 instead of killing just by address. The following filters are available:
 
 * `CLIENT KILL ADDR ip:port`. This is exactly the same as the old three-arguments behavior.
+* `CLIENT KILL LADDR ip:port`. Kill all clients connected to specified local (bind) address.
```

Comment:
need to somewhere mention which version of redis ( >= 6.2 )? this command is supported.
probably do the same for the field.

Reply:
Duplicate of https://github.com/redis/redis-doc/pull/1424#pullrequestreview-519662023

Reply:
Done.